go.mod

```diff
@@ -1,14 +1,12 @@
 module github.com/ngrok/tableroll
 
 require (
-	github.com/go-stack/stack v1.8.0 // indirect
-	github.com/hashicorp/errwrap v1.0.0 // indirect
+	github.com/euank/tableroll v1.0.1-0.20190509205925-1c7855724129
```

Comment:
This is an absolute hack.

We could also have a `third_party/tableroll/v1_0_0` folder where we vendor our old selves I guess. I'm not sure the best solution to regression testing within a single major version in go.